visallo/visallo
# Mark Vertex Children of Compound Nodes as Visible

Mark vertices as visible so that the title of a compound node computes properly on the front end


Testing Instructions:
1. Highlight multiple vertices on a Graph Work Product
2. Right click on the graph and select "Collapse"
3. Verify that the default title for the compound node is a list of the titles from its children
